test/database_api_test.dart

```diff
@@ -472,6 +472,9 @@ void main() {
     test('Close', () async {
       await database.close();
     });
+    test('Delete', () async {
+      await database.delete();
```

Comment:
Can you test that reopening does not contain the same data?

Reply:
Added

Reply:
(this probably doesn't work properly, because in memory dbs vanish on close, but good enough, thanks!)